### PR TITLE
Documentation update for Tutorial-5-7

### PR DESCRIPTION
**Pull request by angranl-flex**
Corresponding Documentation PR: https://github.com/flexcompute-readthedocs/Flow360Documentation/pull/286

### CODE REVIEW

File: examples/tutorials/tutorial_RANS_xv15.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+TutorailRANSXv15.get_files()
+project = fl.Project.from_file(
+    TutorailRANSXv15.mesh_filename,
+    name="Tutorial Time-accurate RANS CFD on XV-15 from Python",
+)
+volume_mesh = project.volume_mesh
+
+with fl.SI_unit_system:
+    rotation_cylinder = volume_mesh["innerRotating"]
```
Comment (benflexcompute) [inline examples/tutorials/tutorial_RANS_xv15.py:16]:
```suggestion
    rotation_zone = volume_mesh["innerRotating"]
```
To not be confused with the cylinder entity concept. It is a zone and then happens to be cylindrical.

File: examples/tutorials/tutorial_RANS_xv15.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                    absolute_tolerance=1e-8,
+                    linear_solver=fl.LinearSolver(max_iterations=25),
+                    DDES=True,
+                    rotation_correction=True,
+                    equation_evaluation_frequency=1,
+                ),
+            ),
+            fl.Rotation(
+                volumes=rotation_cylinder,
+                spec=fl.AngularVelocity(20 * math.pi * fl.u.rad / fl.u.s),
```
Comment (benflexcompute) [inline examples/tutorials/tutorial_RANS_xv15.py:91]: FYI you can use `u.rpm`

File: flow360/component/simulation/models/volume_models.py [hunk 3]
```diff
@@ -55,6 +55,7 @@
 class AngleExpression(SingleAttributeModel):
     """
     :class:`AngleExpression` class for define the angle expression for :py:attr:`Rotation.spec`.
+    The expression of the angle should be specified in the unit of radians.
```
Comment (benflexcompute) [inline flow360/component/simulation/models/volume_models.py:58]:
```suggestion
    The result of the expression is assumed to be in radians.
```

File: examples/tutorials/tutorial_RANS_xv15.py (hunk 1 above)
Comment (angranl-flex) [reply examples/tutorials/tutorial_RANS_xv15.py:16]: OK, updated

File: examples/tutorials/tutorial_dynamic_derivatives.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+# Update the parameters for the unsteady case.
+with fl.SI_unit_system:
+    params.time_stepping = fl.Unsteady(
+        max_pseudo_steps=80,
+        steps=400,
+        step_size=0.01 * 2.0 * math.pi / 20.0 * fl.u.s,
+        CFL=fl.RampCFL(initial=1, final=1e8, ramp_steps=20),
+    )
+    params.models[0].spec = fl.AngleExpression("0.0349066 * sin(0.05877271 * t)")
```
Comment (benflexcompute) [inline examples/tutorials/tutorial_dynamic_derivatives.py:136]: Can you compute the 0.0349066 in expression ad-hoc
or
in documentation explain how the 0.034... is computed?

**Pull request merged**